### ISSUE TITLE
Rust beta crash when running cargo clippy with index out of bound error on travis ci osx

### ISSUE THREAD

**Issue by iamsauravsharma** [rust-lang/rust#64835]
Rust beta currently crash out when cargo clippy in run on travis CI. https://travis-ci.com/iamsauravsharma/cargo-trim/jobs/239695854#L121
If I am correct this issue was fixed by #64774 over nightly but it didn't land over beta rust branch so beta rust is still failing on travis CI OSX.
rust version
`rustc 1.39.0-beta.1 (968967007 2019-09-24)`
cargo version
`cargo 1.39.0-beta (b6c6f685b 2019-09-19)`

**Comment by matthiaskrgr** [comment]
Looks like https://github.com/rust-lang/cargo/issues/7427
Should be fixed once https://github.com/rust-lang/rust/pull/64773 is merged and a new beta is released.

**Comment by jonas-schievink** [comment]
Yeah, duplicate of rust-lang/cargo#7427
